tests/TestCase.php

```diff
@@ -73,6 +73,16 @@ public function createEmptyIndex($indexName, $options = []): Indexes
         return $this->client->getIndex($response['indexUid']);
     }
 
+    public function safeIndexName(string $indexName = 'index'): string
+    {
+        $uidData = random_bytes(16);
+        $uidData[6] = \chr(\ord($uidData[6]) & 0x0F | 0x40);
+        $uidData[8] = \chr(\ord($uidData[8]) & 0x3F | 0x80);
+        $uid = vsprintf('%s%s-%s-%s-%s-%s%s%s', str_split(bin2hex($uidData), 4));
+
+        return $indexName.'_'.$uid;
```

Comment:
maybe `bin2hex(random_bytes(16))` would be enough?

Reply:
Yeah sure, i just used the basic UUIDv4 formatting here but if that is not required the `bin2hex(random_bytes(16))` is way easier 👍 

I guess if UUIDv4 was required it would have been specified in the issue though :D